Source: Hong Hao <oahong@gmail.com>
Upstream: No
Reason: Add zhengma support to tables.coonf
--- data.origin/tables.conf	2009-05-03 22:42:00.000000000 +0800
+++ data/tables.conf	2009-05-15 23:42:41.630146450 +0800
@@ -19,6 +19,19 @@
 符号=zzzz
 
 [码表]
+名称=郑码
+码表=zhengma.mb
+调频=2
+拼音=1
+拼音键=`
+自动上屏=-1
+空码自动上屏=-1
+自动词组=1
+保存自动词组=3
+精确匹配=0
+提示编码=0
+
+[码表]
 名称=五笔拼音
 码表=wbpy.mb
 调频=2
